Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml

```diff
@@ -46,6 +46,8 @@
           Quantity: 3
   - type: Item
     sprite: Objects/Consumable/Food/burger.rsi
+  - type: StaticPrice
+    price: 15
```

Comment:
Food pricings are temporary so this will be reverted at some point.

Reply:
I will remove